### DISCUSSION TITLE
use custom labels and umap coordinates

### DISCUSSION THREAD

**Discussion by sid5427** [stuart-lab/signac#1481] [Q&A] [answered]
Hello,

I am working with some multiome data and was going through the vignettes. 
I have file which has barcode labels as a column, followed by one column which has the cell cluster labels along with umap x and y coordinates in the following columns. Is there a direct way to load these labels into the seurat object after I load the filtered_feature_matrix.h5 & atac_fragments.tsv.gz. I have done something similar while working in archR and scenic, so was wondering if that's possible here...

**Comment by timoast** [comment, marked as answer]
Please see the `AddMetaData()` function and the `meta.data` argument in `CreateSeuratObject()`
